Domains/Smartmetering/smartmeteringwebservices/GetMbusEncryptionKeyStatusByChannel.md

```diff
@@ -0,0 +1,15 @@
+## GetMBusEncryptionKeyStatusByChannel request
+
+### Description
+GetMbusEncryptionKeyStatusByChannel is a request to retrieve the encryption key status of a M-Bus device from an E-meter. The request needs the DeviceIdentification of the gateway device and a channel
```

Comment:
Because of the pronunciation, "an M-Bus device" would be better than "a M-Bus device".
Perhaps finish the sentence with a period.